rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -161,6 +161,24 @@ function hdfs_upload_atomically()
   hdfs dfs -mv $tempFolder ${2} || exit 1
 }
 
+#
+# Prepare the output directory on HDFS
+#
+echo "[INFO] TaskRole:${task_role_no} Task:${PAI_TASK_INDEX}"
```

Comment:
>task_role_no [](start = 24, length = 12)

PAI_OUTPUT_DIR is optional. maybe you should firstly check whether it is being specified or not.

also, can you not use task_role_no? it is likely to be obsoleted later.

Reply:
`outputDir`, which specifies the value of `PAI_OUTPUT_DIR`, is optional in job config json file, but `PAI_OUTPUT_DIR` itself always contains a sound value within job containers. So it is not necessary to do the check.

I saw Can was using this variable in her code. @wangcan0329 : Any alternative we should use?